manticore/core/cpu/arm.py

```diff
@@ -404,23 +408,23 @@ def setFlags(self, **flags):
             instr2 updates all flags in _last_flags except overflow (overflow remains 1 in _last_flags)
             instr2 commits all in _last_flags
             now overflow=1 even though it should still be 0
-        '''
+        """
         unupdated_flags = self._last_flags.viewkeys() - flags.viewkeys()
         for flag in unupdated_flags:
             flag_name = 'APSR_{}'.format(flag)
             self._last_flags[flag] = self.regfile.read(flag_name)
         self._last_flags.update(flags)
 
-    def commitFlags(self):
+    def commit_flags(self):
         # XXX: capstone incorrectly sets .update_flags for adc
         if self.instruction.mnemonic == 'adc':
             return
         for flag, val in self._last_flags.iteritems():
             flag_name = 'APSR_{}'.format(flag)
             self.regfile.write(flag_name, val)
 
-    def _Shift(cpu, value, _type, amount, carry):
-        assert(_type > cs.arm.ARM_SFT_INVALID and _type <= cs.arm.ARM_SFT_RRX_REG)
+    def _shift(cpu, value, _type, amount, carry):
```

Comment:
The reason we originally had it named like this was to closely match the arm manual pseudocode; however I think this change is a good thing to make the codebase more pythonic. Can you please add a docstring which states "See Shift() and Shift_C() in the ARM manual"?

![image](https://user-images.githubusercontent.com/2467355/36320651-e132f95e-1314-11e8-83c1-c9c447874fdf.png)

Reply:
@mossberg added